recipes/lora_finetune_distributed.py

```diff
@@ -541,9 +541,12 @@ def train(self) -> None:
                 if (
                     self.total_training_steps % self._log_peak_memory_every_n_steps == 0
                     and self._is_rank_zero
+                    and self._device == torch.device("cuda")
```

Comment:
Do we really need this one too? For distributed tests they should only run on GPU

Reply:
mm yeah I can remove this check for distributed recipes